src/QZXing.cpp

```diff
@@ -79,6 +79,8 @@ QZXing::QZXing(QZXing::DecoderFormat decodeHints, QObject *parent) : QObject(par
 
 #ifdef QZXING_QML
 
+#ifndef DISABLE_LIBRARY_FEATURES
```

Comment:
This should be removed as this macro refers to the integration method of the library, not to the functionality itself. 

Thus the same comment applies to the closing of the ifdef.

Reply:
But its needed on static builds and not on dynamic builds.
I thought that macro was to check if no dynamic build was built.